src/main/scala/everywhere/e300artydevkit/E31-Core.scala

```diff
@@ -0,0 +1,51 @@
+package sifive.freedom.everywhere.e300artydevkit
```

Comment:
Can you make this all in the same Config.scala file instead of creating a separate one?